### ISSUE TITLE
AWS instance user_data: invalid BASE64 encoding

### ISSUE THREAD

**Issue by Falx** [hashicorp/terraform#1196]
When using terraform to create 3 AWS instances with a file as user_data. The following error occurs:

`Error launching source instance: Invalid BASE64 encoding of user data`

My tf file is:
```yml
resource "aws_instance" "analytics" {
    ami = "..."
    instance_type = "..."
    count = "3"
    subnet_id = "..."
    security_groups = ["..."]
    key_name = "..."

    // This loads the cloud config on every instance.
    user_data = "${file("coreos/cloud-config.yml")}"

    block_device {...}

    connection {...}

    provisioner "file" {...}

    provisioner "remote-exec" {...}
}

```

Asked a question in IRC forum, they told me to post the issue..

**Comment by catsby** [comment]
Thanks for reporting this! I was the one that requested you post here :smile: 

I have a fix, I'll get it rolled out as soon as possible.

**Comment by Falx** [comment]
Nice, thanks for the quick reaction ;-)

**Comment by catsby** [comment]
Fixed in #1197

**Comment by catsby** [comment]
#1197 is merged, thanks again @Falx
